src/Workspaces/Core/Portable/Remote/RemoteHostClient.cs

```diff
@@ -26,6 +26,14 @@ protected RemoteHostClient(Workspace workspace)
 
         public event EventHandler<bool> StatusChanged;
 
+        /// <summary>
+        /// Return an unique string per client.
```

Comment:
Isn't it unique per host?

Reply:
client is tied to 1 process. when client go away process go away as well.

Reply:
let me add some helper for you actually.